spec/controller_services/shopping_list_items_controller/create_service_spec.rb

```diff
@@ -33,12 +33,13 @@
         end
 
         it 'adds a list item to the given list' do
-          expect { perform }.to change(shopping_list.list_items, :count).from(0).to(1)
+          expect { perform }
+            .to change(shopping_list.list_items, :count).from(0).to(1)
         end
 
         it 'assigns the correct values' do
-          params_with_string_keys = {}
-          params.each { |key, value| params_with_string_keys[key.to_s] = value }
+          params_with_string_keys                                     = {}
```

Comment:
It probably shouldn't actually be like this but it's a bug in Rubocop and the last thing I want is to already have `rubocop:disable` comments all over my code already, so I'll live with it.